modules/components/src/Aggs/BooleanAgg.js

```diff
@@ -0,0 +1,122 @@
+import React, { Component } from 'react';
+import AggsWrapper from './AggsWrapper.js';
+import { css } from 'emotion';
+import {
+  toggleSQON,
+  inCurrentSQON,
+  replaceSQON,
+  removeSQON,
+} from '../SQONView/utils';
+import './BooleanAgg.css';
+
+export default ({
+  field,
+  buckets,
+  handleValueClick = () => {},
+  isActive = () => false,
+  ...rest
+}) => {
+  const trueBucket = buckets.find(({ key }) => key === '1');
+  const falseBucket = buckets.find(({ key }) => key === '0');
+  const dotField = field.replace(/__/g, '.');
+
+  const createSqonValidator = isTrue => ({ sqon }) =>
+    inCurrentSQON({
+      dotField,
+      value: isTrue,
+      currentSQON: sqon,
+    });
+
+  const isTrueActive = isActive({
+    defaultEvaluator: createSqonValidator(true),
```

Comment:
I don't think you need `defualtEvaluator` or `createSqonValidator`. the term and boolean agg do the same thing for `isActive` so just let the parent handle this.